chain/chain/src/stateless_validation/chunk_validation.rs

```diff
@@ -369,11 +381,19 @@ pub fn pre_validate_chunk_state_witness(
 
     let current_protocol_version =
         epoch_manager.get_epoch_protocol_version(&state_witness.epoch_id)?;
-    if !checked_feature!(
+    let transaction_validity_check_passed = if checked_feature!(
         "protocol_feature_relaxed_chunk_validation",
         RelaxedChunkValidation,
         current_protocol_version
     ) {
+        if last_chunk_block.header().is_genesis() {
+            vec![true; state_witness.transactions.len()]
```

Comment:
I had to add this to make test that start at the genesis to pass. There is no "prev_block_header" in this instance. The code further below that relies on the previous block has a similar special case for the genesis.

This vector currently ends up not being used, but if it did, it seemed to me that considering all transactions "valid" is reasonable enough: the transactions can't be much older than any useful `transaction_validity_period`. On the other hand if there are additional checks added to `transaction_validity_check` in the future, this might no longer hold… So this is not a particularly resilient piece of code in my eyes.